versions.bzl

```diff
@@ -0,0 +1,5 @@
+KOTLIN_VERSION = "1.3.31"
```

Comment:
Optional: you could export a struct containing all these versions, rather than having a massive list of constants. It'll make it easier to import in the WORKSPACE file:

```
load(":versions.bzl", "versions")

versions.auto_value
versions.atomicfu
```